web/src/main/java/org/bookkeepervisualmanager/api/resources/LedgersResource.java

```diff
@@ -101,4 +101,30 @@ private Integer convertParam(String s) {
         }
     }
 
+    /**
+     * Split a string into a Long list. If any value is invalid, return empty list
+     *
+     * @param s
+     * @return List
+     */
+    private List<Long> convertParamLongList(String s) {
+        if (s == null || s.trim().isEmpty()) {
+            return null;
+        }
+        List<Long> res = new ArrayList<>();
+        for (String _s : s.split(",")) {
+            _s = _s.trim();
+            if (!_s.isEmpty()) {
+                try {
+                    Long l = Long.parseLong(_s);
+                    res.add(l);
+                } catch (NumberFormatException ex) {
+                    res.clear();
```

Comment:
in this case we won't filter out any id.
Proposals:
- ignore the value
- throw an error (and handle it upstream)

I am happy with any of the two choices

Reply:
Method now throws a NumberFormatException handled to check errors.